packages/radix-ui-themes/src/styles/tokens/color.css

```diff
@@ -157,19 +153,38 @@
 @import '@radix-ui/colors/sand-alpha.css';
 @import '@radix-ui/colors/sand-dark-alpha.css';
 
-/* overlays */
 @import '@radix-ui/colors/black-alpha.css';
 @import '@radix-ui/colors/white-alpha.css';
 
-.radix-themes {
-  /* text color */
-  color: var(--gray-12);
+/* * * * * * * * * * * * * * * * * * * */
+/*                                     */
+/*           Semantic colors           */
+/*                                     */
+/* * * * * * * * * * * * * * * * * * * */
```

Comment:
Do you mind big comments to denote sections @benoitgrelard? I tend to miss the comments we have in files like this otherwise